### ISSUE TITLE
To split: MotoGP™22

### ISSUE THREAD

**Issue by HarkTu** [IsThereAnyDeal/IsThereAnyDeal-Issues#3299]
https://store.steampowered.com/app/1710580/
shows price history for another game
https://store.steampowered.com/app/240600/

**Comment by candela97** [comment]
Yep, https://store.steampowered.com/app/1710580/ links to https://isthereanydeal.com/game/motogpiiii/info/, which contains mixed entries for both MotoGP™22 and MotoGP™13 that should be split.
